.github/workflows/deploy-lts-prow.yaml

```diff
@@ -39,9 +42,10 @@ jobs:
         id: bicep
         uses: azure/arm-deploy@v2
         with:
-          subscriptionId: ${{ vars.AZURE_SUBSCRIPTION }}
```

Comment:
why we have this as AZURE_SUBSCRIPTION before?

Reply:
yep, that's a duplicate.